src/cplus_plugin/tasks.py

```diff
@@ -101,8 +101,8 @@ def get_masking_layers(self):
             Settings.MASK_LAYERS_PATHS, default=None
         )
         masking_layers = masking_layers_paths.split(",") if masking_layers_paths else []
-
         masking_layers.remove("") if "" in masking_layers else None
+        masking_layers = [ml.replace('.shp', '.zip') for ml in masking_layers]
```

Comment:
@zamuzakki, if this line is used before creating json for the API, I think it's better to move this line to upload_layers method. The reason is that the ScenarioAnalysisTask class is also used for running the analysis in local machine.